migrations/V241__Add_Key_Contact_Tables.sql

```diff
@@ -0,0 +1,65 @@
+ALTER TYPE TABLE_NAME ADD VALUE 'key_contact_category';
+
+-- Create table for key contact category
+CREATE TABLE IF NOT EXISTS  key_contact_category (
+    id UUID PRIMARY KEY NOT NULL,
+    category ZERO_STRING NOT NULL,
+
+    --META DATA
+    created_by UUID NOT NULL REFERENCES user_account(id),
+    created_dts TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by UUID REFERENCES user_account(id),
+    modified_dts TIMESTAMP WITH TIME ZONE
+);
+
+COMMENT ON TABLE key_contact_category IS
+'Table for storing key contact categories related to subject matter experts.';
+
+ALTER TYPE TABLE_NAME ADD VALUE 'key_contact';
+
+-- Create table for key contact
+CREATE TABLE IF NOT EXISTS  key_contact (
+    id UUID PRIMARY KEY NOT NULL,
+    user_id UUID REFERENCES user_account(id) ON DELETE CASCADE,
+    
+    subject_area ZERO_STRING NOT NULL,
+    subject_category_id UUID NOT NULL REFERENCES key_contact_category(id) ON DELETE CASCADE,
+
+    is_team BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
potential nitpick: is_team is kind of redundant. The check constraint already enforces that a contact is either user-based or mailbox-based. You can derive is_team from user_id IS NULL. Consider removing this column to avoid potential inconsistency, or add a constraint ensuring is_team = (user_id IS NULL).